tests/libnet/ping.go

```diff
@@ -43,7 +43,7 @@ func PingFromVMConsole(vmi *v1.VirtualMachineInstance, ipAddr string, args ...st
 	}
 
 	if len(args) == 0 {
-		args = []string{"-c 1", "-w 5"}
+		args = []string{"-c 5", "-w 5"}
```

Comment:
You need to increase now the deadline (-w) by at least double. A value of 10 should work.
Note: The default interval between pings is 1sec.

Reply:
Done.